src/features/cartographie/components/markers/lieu-mediation-numerique-markers/lieu-mediation-numerique-markers.component.html

```diff
@@ -19,6 +19,7 @@
         ·
         <small>{{ lieuMediationNumerique.status.limite }}</small>
       </div>
+      <h5 class="my-1" *ngIf="lieuMediationNumerique.prive">Ce lieu n'accueil pas de publique</h5>
```

Comment:
```suggestion
      <span class="my-1 fs-4" *ngIf="lieuMediationNumerique.prive">Ce lieu n'accueille pas de publique</span>
```
Les titres doivent suivre une séquence logique dans la hérarchie de la page pour que l'accessibilité soit valide. Mettre des h5 pour toutes les popup de la carte risque de poser des soucis